source/includes/read/specify-queries.scala

```diff
@@ -0,0 +1,77 @@
+import org.mongodb.scala._
+import org.mongodb.scala.model.Filters._
+import org.mongodb.scala.model.Projections._
+import org.mongodb.scala.model.Sorts._
+import scala.concurrent.Await
+import scala.concurrent.duration._
+import scala.util.{Failure, Success}
+import scala.concurrent.ExecutionContext.Implicits.global
+
+object Retrieve {
+  def main(args: Array[String]): Unit = {
+    // start-setup
+    val uri: String = "<connection string>"
+    val client: MongoClient = MongoClient(uri)
+    val database: MongoDatabase = client.getDatabase("db")
+    val collection: MongoCollection[Document] = database.getCollection("fruits")
+
+    // Inserts documents representing fruits
+    val fruits: Seq[Document] = Seq(
+        Document("_id" -> 1, "name" -> "apples", "qty" -> 5, "rating" -> 3, "color" -> "red", "type" -> Seq("fuji", "honeycrisp")),
+        Document("_id" -> 2, "name" -> "bananas", "qty" -> 7, "rating" -> 4, "color" -> "yellow", "type" -> Seq("cavendish")),
+        Document("_id" -> 3, "name" -> "oranges", "qty" -> 6, "rating" -> 2, "type" -> Seq("naval", "mandarin")),
+        Document("_id" -> 4, "name" -> "pineapples", "qty" -> 3, "rating" -> 5, "color" -> "yellow")
+    )
+
+    collection.insertMany(fruits)
```

Comment:
This doesn't do anything apart from create an `Observable` it must be subscribed to, to actually write some data.